q1.java

```diff
@@ -0,0 +1,82 @@
+
+import java.util.HashMap;
+
+public class q1 {
+	
+	public static void computeCharCount(String s, HashMap<Character, Integer> s_charCount){
+

```

Comment:
As you are not using the existing contents of s_charCount. Instantiating it within the computeCharCount would be better than passing it in. Then you are passing in a String, and returning the Mapping